unc-cs-sg/myhealthed-admin
# Add LinkDemo page to master

_Overview_:
* adds LinkDemo page to test front-end routes (for Link List View)

_Additional changes in last commit_:
* Replaced `fetch` with `axios` (note that it auto-transforms JSON data into native JS objects , but provides data in the response.data field of the response object)
* Added a bunch of comments with links to documentation
* Change d2-space indents to 4-space indents
* Clean up Promises and code style 
* Delete an unused routes.js page (it'll come back later, but for now, removing it so it's not confusing)
